staging/src/k8s.io/legacy-cloud-providers/azure/azure.go

```diff
@@ -487,6 +502,7 @@ func (az *Cloud) InitializeCloudFromConfig(config *Config, fromSecret bool) erro
 		ShouldOmitCloudProviderBackoff: config.shouldOmitCloudProviderBackoff(),
 		Backoff:                        &retry.Backoff{Steps: 1},
 	}
```

Comment:
could you refactor the following code changes as part of azClientConfig initilization? e.g. a new function could be added `getAzureClientConfig()` and the token assignment could be all inside it.

Reply:
Make sense. Updated. Thank you.